source/Engine.cpp

```diff
@@ -1488,7 +1488,12 @@ void Engine::CalculateStep()
 	Point newCenterVelocity;
 	if(flagship)
 	{
-		newCenter = flagship->Position();
+		if(flagship->IsThrusting())
+			offsetMultiplier = offsetMultiplier >= 1 ? 1.0 : offsetMultiplier + 0.0025;
+		else
+			offsetMultiplier = offsetMultiplier <= 0 ? 0.0 : offsetMultiplier - 0.0025;
```

Comment:
Compare offsetMultiplier with 0. instead of 0 to avoid a needless convertion from int to double

Reply:
done